python/whylogs/core/resolvers.py

```diff
@@ -67,6 +110,14 @@ def resolve(self, name: str, why_type: DataType, column_schema: ColumnSchema) ->
         result: Dict[str, Metric] = {}
         for m in metrics:
             result[m.name] = m.zero(column_schema.cfg)
+
+        if self.additional_metrics:
+            additional_metrics_result = self.resolve_additional_standard_metrics(
+                result=result, name=name, why_type=why_type, column_schema=column_schema
+            )
+            for key in additional_metrics_result.keys():
+                result[key] = additional_metrics_result[key]
```

Comment:
you can do:
```suggestion
            result.update(additional_metrics_result)
```

Reply:
thanks! much better, added in the latest results